include/lsst/daf/base/DateTime.h

```diff
@@ -63,15 +63,15 @@ class DateTime {
     explicit DateTime(double date, DateSystem system = MJD, Timescale scale = TAI);
     DateTime(int year, int month, int day, int hr, int min, int sec,
              Timescale scale = TAI);
-    explicit DateTime(std::string const& iso8601);
+    explicit DateTime(std::string const& iso8601, Timescale scale);
```

Comment:
No default? Does the name iso8601 still make sense for all possible values of Timescale?